addusers.py

```diff
@@ -467,6 +496,17 @@ def parse_rows(rows, select_user=None):
                    "You may need to close multiple tickets.").format(
                        len(copy_index), args.user)
         copy_rows = [r for r in rows if rows.index(r) in copy_index]
+        max_length = len(header)
+        user_header_column = header.index(email_column)
+        new_project_header_column = header.index(project_name_column)
```

Comment:
Are the variables set in lines 501 and 502 used anywhere?  if not, get rid of them.